libraries/standard/http/test/README.md

```diff
@@ -0,0 +1,17 @@
+# Instructions for running the tests:
```

Comment:
Will add this to https://w.amazon.com/bin/view/AWS/Mobile/IoT/Treadstone/LabsExitCriteria in the future